src/redux/reducers/cellSets/initialState.js

```diff
@@ -4,6 +4,7 @@ const initialState = {
   selected: [],
   loading: true,
   error: false,
+  updatingClustering: false,
```

Comment:
It would be better to reuse `loading` - the purpose of it is to indicate if any parts of the data is currently under a change. Higher-level components check what is the state of 'loading' and if it is true, they know that they have to show the spinner. Is there any reason why we can't use it in here?

Reply:
I think we need two separate properties.

If someone triggers a `loadCellSets` action while the clustering is being updated then we will probably get this new action to finish before `updateCellSetsClustering` does, which will set the state to `CELL_SETS_LOADED`.

Reply:
Yeah, I understand now why you decided to add it. However, looking at the action for `loadCellSets`, specifically at this piece of code:

```
if ((!loading && !error) || updatingClustering) {
    return null;
}
```
This means that if cell sets is in loading state, the action will return null anyway. Therefore, if we made `updateCellSetsClustering` use `loading` instead of `updatingClustering`, it will technically work. Or am I missing something?

Reply:
It returns null if cell sets is **not** in loading state. I think the reason that this is so is because the cell sets are in loading state when they begin, so that's when we would probably want to load (and avoid loading again).

I didn't want to change that logic so that's why I added this.

Reply:
I see, somehow i skipped the `not` bit. Ok, I am convinced, sounds like the most sensible thing to do is to have this `updatingClustering` boolean